docs/how-to/vector-stores/hybrid-search.md

```diff
@@ -0,0 +1,169 @@
+# Hybrid Search with Vector Stores
+
+Hybrid search combines multiple retrieval methods to improve search quality. This guide explains how to implement hybrid search using Ragbits vector stores.
+
+## What is Hybrid Search?
+
+Hybrid search combines different search approaches, typically:
+- **Dense retrieval**: Using dense embeddings for semantic similarity
+- **Sparse retrieval**: Using sparse embeddings for lexical/keyword matching
+
+By combining these approaches, hybrid search can overcome the limitations of each individual method.
+
+## Implementing Hybrid Search in Ragbits
+
+Ragbits provides several strategies for combining results from different vector stores:
+
+### 1. Set Up Multiple Vector Stores
+
+First, create vector stores with different embedding types:
+
+```python
+from ragbits.core.embeddings.dense import DenseEmbedder
+from ragbits.core.embeddings.sparse import SparseEmbedder
+from ragbits.core.vector_stores.in_memory import InMemoryVectorStore
+from ragbits.core.vector_stores.base import EmbeddingType
+
+# Dense embedder for semantic search
+dense_embedder = DenseEmbedder(model_name="sentence-transformers/all-MiniLM-L6-v2")
+dense_store = InMemoryVectorStore(
+    embedder=dense_embedder,
+    embedding_type=EmbeddingType.TEXT
+)
+
+# Sparse embedder for lexical search
+sparse_embedder = SparseEmbedder(vocabulary_size=10000)
+sparse_store = InMemoryVectorStore(
+    embedder=sparse_embedder,
+    embedding_type=EmbeddingType.TEXT
+)
+```
+
+### 2. Choose a Hybrid Retrieval Strategy
+
+Ragbits provides several strategies for combining results:
+
+```python
+from ragbits.core.vector_stores.hybrid_strategies import (
+    OrderedHybridRetrivalStrategy,
+    ReciprocalRankFusion,
+    DistributionBasedScoreFusion
+)
+
+# Choose one of the available strategies
+strategy = ReciprocalRankFusion()
+```
+
+Available strategies:
+- **OrderedHybridRetrivalStrategy**: Orders results by score and deduplicates them
+- **ReciprocalRankFusion**: Combines results based on their ranks rather than raw scores
+- **DistributionBasedScoreFusion**: Normalizes scores based on their distribution before combining
+
+### 3. Perform Hybrid Search
```

Comment:
The example doesn't use the `HybridSearchVectorStore` - so it doesn't explain the recommended way to perform hybrid search in ragbits